### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by ChristopherWinterZF**
First SSI doc artefacts

### CODE REVIEW

File: docs/architecture/cx-3-2/3. Verifiable Credentials/CX-Credentials/Standardized CX-Credential.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+```json
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/jws-2020/v1",
+         "https://raw.githubusercontent.com/catenax-ng/product-core-schemas/main/businessPartnerData"
+    ],
+    "id": "http://example.edu/credentials/58473",
+    "issuer": "<did>",
+    "type": ["VerifiableCredential", "DismantlerCredentialCX"],
```
Comment (jimmarino) [inline docs/architecture/cx-3-2/3. Verifiable Credentials/CX-Credentials/Standardized CX-Credential.md:86]: Type should be MembershipCredential

**Review by jimmarino — COMMENTED** [review]
Are the CX suffixes on the VC types correct?

**Comment by jimmarino** [comment]
I also think the Summary VP example is incorrect. The property should be "verifiableCredential" not the plural form:

```json
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1"
  ],
  "type": "VerifiablePresentation",
  "verifiableCredential": [
    {
      "@context": [
        "https://www.w3.org/2018/credentials/v1"
      ],
      "id": "urn:uuid:12345678-1234-1234-1234-123456789abc",
      "type": [
        "VerifiableCredential",
        "SummaryCredential"
      ],
      "issuer": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
      "issuanceDate": "2023-06-02T12:00:00Z",
      "expirationDate": "2022-06-16T18:56:59Z",
      "credentialSubject": {
        "id": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
        "holderIdentifier": "BPN of holder",
        "type": "Summary-List",
        "name": "CX-Credentials",
        "items": [
          "cx-active-member",
          "cx-dismantler",
          "cx-pcf",
          "cx-sustainability",
          "cx-quality",
          "cx-traceability",
          "cx-behavior-twin",
          "cx-bpn"
        ],
        "contract-templates": "https://public.catena-x.org/contracts/"
      },
      "proof": {
        "type": "Ed25519Signature2018",
        "created": "2023-06-02T12:00:00Z",
        "proofPurpose": "assertionMethod",
        "verificationMethod": "did:web:example.com#key-1",
        "jws": "eyJhbGciOiJFZERTQSJ9.eyJpYXQiOjE2MjM1NzA3NDEsImV4cCI6MTYyMzU3NDM0MSwianRpIjoiMTIzNDU2NzgtMTIzNC0xMjM0LTEyMzQtMTIzNDU2Nzg5YWJjIiwicHJvb2YiOnsiaWQiOiJkaWQ6d2ViOmV4YW1wbGUuY29tIiwibmFtZSI6IkJlaXNwaWVsLU9yZ2FuaXNhdGlvbiJ9fQ.SignedExampleSignature"
      }
    }
  ]
} 

```

**Comment by jimmarino** [comment]
Over the weekend we added the following:

- [VC Technical Guidelines for Tractus-X](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/credentials/tx.vc.guidelines.md)

- [Summary VC Specification](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/credentials/summary/summary.vc.md)

I'd like to align our efforts around the items in those documents. In particular, the second contains issues we need to address related to the Summary VP.

File: docs/architecture/cx-3-2/3. Verifiable Credentials/CX-Credentials/Standardized CX-Credential.md (hunk 1 above)
Comment (maximilianschmidt-zf) [reply docs/architecture/cx-3-2/3. Verifiable Credentials/CX-Credentials/Standardized CX-Credential.md:86]: This is the Dismantler Credential why should the Type be Membership Credential?

**Comment by maximilianschmidt-zf** [comment]
> I also think the Summary VP example is incorrect. The property should be `verifiableCredential`, not the plural form:
> 
> ```json
> {
>   "@context": [
>     "https://www.w3.org/2018/credentials/v1"
>   ],
>   "type": "VerifiablePresentation",
>   "verifiableCredential": [
>     {
>       "@context": [
>         "https://www.w3.org/2018/credentials/v1"
>       ],
>       "id": "urn:uuid:12345678-1234-1234-1234-123456789abc",
>       "type": [
>         "VerifiableCredential",
>         "SummaryCredential"
>       ],
>       "issuer": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
>       "issuanceDate": "2023-06-02T12:00:00Z",
>       "expirationDate": "2022-06-16T18:56:59Z",
>       "credentialSubject": {
>         "id": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
>         "holderIdentifier": "BPN of holder",
>         "type": "Summary-List",
>         "name": "CX-Credentials",
>         "items": [
>           "cx-active-member",
>           "cx-dismantler",
>           "cx-pcf",
>           "cx-sustainability",
>           "cx-quality",
>           "cx-traceability",
>           "cx-behavior-twin",
>           "cx-bpn"
>         ],
>         "contract-templates": "https://public.catena-x.org/contracts/"
>       },
>       "proof": {
>         "type": "Ed25519Signature2018",
>         "created": "2023-06-02T12:00:00Z",
>         "proofPurpose": "assertionMethod",
>         "verificationMethod": "did:web:example.com#key-1",
>         "jws": "eyJhbGciOiJFZERTQSJ9.eyJpYXQiOjE2MjM1NzA3NDEsImV4cCI6MTYyMzU3NDM0MSwianRpIjoiMTIzNDU2NzgtMTIzNC0xMjM0LTEyMzQtMTIzNDU2Nzg5YWJjIiwicHJvb2YiOnsiaWQiOiJkaWQ6d2ViOmV4YW1wbGUuY29tIiwibmFtZSI6IkJlaXNwaWVsLU9yZ2FuaXNhdGlvbiJ9fQ.SignedExampleSignature"
>       }
>     }
>   ]
> } 
> ```

If we change it we are not standard compliant as the property is from the w3c implementation guide which refers to it in singular - https://www.w3.org/TR/vc-imp-guide/#presentations also in the other standards it is in singular - https://www.w3.org/TR/vc-data-model/#presentations-0

**Comment by jimmarino** [comment]
> > I also think the Summary VP example is incorrect. The property should be `verifiableCredential`, not the plural form:
> > ```json
> > {
> >   "@context": [
> >     "https://www.w3.org/2018/credentials/v1"
> >   ],
> >   "type": "VerifiablePresentation",
> >   "verifiableCredential": [
> >     {
> >       "@context": [
> >         "https://www.w3.org/2018/credentials/v1"
> >       ],
> >       "id": "urn:uuid:12345678-1234-1234-1234-123456789abc",
> >       "type": [
> >         "VerifiableCredential",
> >         "SummaryCredential"
> >       ],
> >       "issuer": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
> >       "issuanceDate": "2023-06-02T12:00:00Z",
> >       "expirationDate": "2022-06-16T18:56:59Z",
> >       "credentialSubject": {
> >         "id": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
> >         "holderIdentifier": "BPN of holder",
> >         "type": "Summary-List",
> >         "name": "CX-Credentials",
> >         "items": [
> >           "cx-active-member",
> >           "cx-dismantler",
> >           "cx-pcf",
> >           "cx-sustainability",
> >           "cx-quality",
> >           "cx-traceability",
> >           "cx-behavior-twin",
> >           "cx-bpn"
> >         ],
> >         "contract-templates": "https://public.catena-x.org/contracts/"
> >       },
> >       "proof": {
> >         "type": "Ed25519Signature2018",
> >         "created": "2023-06-02T12:00:00Z",
> >         "proofPurpose": "assertionMethod",
> >         "verificationMethod": "did:web:example.com#key-1",
> >         "jws": "eyJhbGciOiJFZERTQSJ9.eyJpYXQiOjE2MjM1NzA3NDEsImV4cCI6MTYyMzU3NDM0MSwianRpIjoiMTIzNDU2NzgtMTIzNC0xMjM0LTEyMzQtMTIzNDU2Nzg5YWJjIiwicHJvb2YiOnsiaWQiOiJkaWQ6d2ViOmV4YW1wbGUuY29tIiwibmFtZSI6IkJlaXNwaWVsLU9yZ2FuaXNhdGlvbiJ9fQ.SignedExampleSignature"
> >       }
> >     }
> >   ]
> > } 
> > ```
> 
> If we change it we are not standard compliant as the property is from the w3c implementation guide which refers to it in singular - https://www.w3.org/TR/vc-imp-guide/#presentations also in the other standards it is in singular - https://www.w3.org/TR/vc-data-model/#presentations-0

Please also see [Summary VC Specification](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/credentials/summary/summary.vc.md) which adds a specification for the VC and lists issues with it that we need to address.

File: docs/architecture/cx-3-2/6. Technical Debts/Technical Debts.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* CX-Credentials are not consistent
+
+## SSI Lib
+  
+  * No complete JsonWebSignature2020. Only ED22519 is supported.
+  * No validation for JsonWebSignature2020 with RSA key.
+  * No Security valdition only Sercurity Assesment. No attack vectors are tested. 
+  * ...
+
+## EDC
```
Comment (jimmarino) [inline docs/architecture/cx-3-2/6. Technical Debts/Technical Debts.md:36]: The HTTP header is not a limitation of the EDC

**Review by maximilianschmidt-zf — APPROVED** [review]

**Comment by paullatzelsperger** [comment]
@ChristopherWinterZF the "new" initial commit PR is here: #12

**Pull request closed without merging**